src/desktop/apps/auction/components/DOM.js

```diff
@@ -38,15 +40,43 @@ class DOM extends Component {
     this.$body = this.$("body")
     this.$body.find(".Sidebar").on("click", ".artsy-checkbox", scrollToTop)
     this.$registerBtn = this.$body.find(".js-register-button")
-    this.$registerBtn.on("click", this.handleRegister)
+    this.$registerBtn.on("click", this.handleRegister(false))
   }
 
   removeEventListeners() {
     this.$body.off("click")
-    this.$registerBtn.off("click", this.handleRegister)
+    this.$registerBtn.off("click")
+  }
+
+  showModal = (type, delay = true) => {
+    // Showing modal immediately requires a delay due to jockeying with
+    // the backbone-based ClockView inside Banner
+    setTimeout(
+      () => {
+        this.props.dispatch(showModal(type))
+      },
+      delay ? 1000 : 0
```

Comment:
This is a bit weird -- if it requires `setTimeout`, best to just set it and be done and add a `// FIXME: Remove timeout when we remove backbone`